set.mm

```diff
@@ -121476,8 +121476,23 @@ subset of complex numbers ( ~ nnsscn ), in contrast to the more elementary
       PVIVHVAEUIARFZUQRFZVHVALZVGASUQSVJVKKRFVLTAUQKUAUBUCNUDUEUFUGUH $.
 
     $( Closure of multiplication of positive integers.  (Contributed by NM,
-       12-Jan-1997.) $)
+       12-Jan-1997.)  Reduce dependencies on axioms.  (Revised by Steven
```

Comment:
Can you be more explicit ? I.e.:
```
...).  Remove dependency on ~ ax-A , ~ ax-B , ~ ax-C .  (Revised by...
```

Reply:
And, while on this topic: in ~1t1e1ALT, can you write "Alternate proof" instead of "Alternate version" (since this is the same statement), and can you write:
```
using a different set of axioms (add ~ ax-mulrcl , ~ ax-i2m1 , ~ ax-1ne0 , ~ ax-rrecex and
remove ~ ax-resscn , ~ ax-mulcom , ~ ax-mulass , ~ ax-distr ).
```
This may help other people studying the axiomatizations of $\mathbb{C}$.

Reply:
ax-mulcom and ax-mulass are not referenced in the new proof